app/src/main/java/com/github/muellerma/prepaidbalance/ui/MainActivity.kt

```diff
@@ -130,18 +141,32 @@ class MainActivity : AbstractBaseActivity(), SwipeRefreshLayout.OnRefreshListene
 
     private fun exportAsCsv() {
         launch {
-            val content = buildCsv()
+//            val content = buildCsv()
+//
+//            try {
+//                val filename = "prepaid-balance-${System.currentTimeMillis()}.csv"
+//                writeToFileInDownloads(content, filename)
+//                showSnackbar(getString(R.string.export_saved_file, filename))
+//                return@launch
+//            } catch (e: Exception) {
+//                Log.e(TAG, "Error saving file", e)
+//            }
+//
+//            showSnackbar(R.string.export_error_saving_file)
```

Comment:
Please remove this.

Reply:
I was preventing myself from removing any piece of your code. That was why I commented it and not delete it.